fuzzing_tc/common/pool.py

```diff
@@ -23,13 +24,13 @@
         "container": str,
         "cores_per_task": int,
         "cpu": str,
-        "cycle_time": str,
-        "disk_size": str,
+        "cycle_time": (int, str),
+        "disk_size": (int, str),
         "imageset": str,
         "macros": dict,
-        "max_run_time": str,
+        "max_run_time": (int, str),
         "metal": bool,
-        "minimum_memory_per_core": str,
+        "minimum_memory_per_core": (float, str),
```

Comment:
Why is this `(float, str)` and not `(int, str)`?

Reply:
I thought it made sense to allow fractional RAM sizes, so you can do `"3.5g"` or whatever.  `disk_size` could also be `float`, but it seems unlikely to be used.

The other `int` fields are for `cores_per_task` and `tasks`, which make sense as whole numbers. `cycle_time` and `max_run_time` are in seconds, so don't need to be divided further.

Reply:
If you are already handling the string "3.5g" would it make sense to convert that value to MBs (since this is the smallest unit we will deal with) and store that value as an int?

Reply:
That's an easy change to make, however I think it makes it more error prone. If you missed the "g" it would default to 4MB instead of 4GB. The whole point of allowing the string suffixes is to avoid having to do math.